sinatra-soap.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rack-test"
-  spec.add_development_dependency "debugger"
+  #spec.add_development_dependency "debugger"
```

Comment:
It's only for ruby 1.9.*, I think there was some installation issues with ruby 2.4